pom.xml

```diff
@@ -2762,8 +2762,10 @@
           <failOnViolation>true</failOnViolation>
           <includeTestSourceDirectory>false</includeTestSourceDirectory>
           <failOnWarning>false</failOnWarning>
-          <sourceDirectory>${basedir}/src/main/scala</sourceDirectory>
-          <testSourceDirectory>${basedir}/src/test/scala</testSourceDirectory>
+          <sourceDirectories>${basedir}/src/main/scala
+            <dir>${basedir}/src/main/scala</dir>
+            <dir>${basedir}/src/test/scala</dir>
```

Comment:
why did we add the test directory in here?  We have includeTestSourceDirectory=false for some reason above.  if we just turn that on do things work or does it cause some other problems?

Reply:
oops never mind, looks like you had comment already

Reply:
Oh, I missed that property. I'll turn on that.